src/Npgsql.NodaTime/TimestampTzHandler.cs

```diff
@@ -13,26 +13,45 @@ public class TimestampTzHandlerFactory : NpgsqlTypeHandlerFactory<Instant>
     {
         // Check for the legacy floating point timestamps feature
         public override NpgsqlTypeHandler<Instant> Create(PostgresType postgresType, NpgsqlConnection conn)
-            => conn.HasIntegerDateTimes
-                ? new TimestampTzHandler(postgresType)
-                : throw new NotSupportedException($"The deprecated floating-point date/time format is not supported by {nameof(Npgsql)}.");
+        {
+            var csb = new NpgsqlConnectionStringBuilder(conn.ConnectionString);
```

Comment:
Any reason to be parsing the connection string here? [See how the BCL TimestampHandlerFactory does it](https://github.com/npgsql/npgsql/blob/90366685b88456446988ca087ec9c1a53e0613bd/src/Npgsql/TypeHandlers/DateTimeHandlers/TimestampHandler.cs#L25)

Reply:
Ah I see, because NpgsqlConnector is internal... OK for now I guess...

Reply:
Yeah, same as https://github.com/npgsql/npgsql/blob/dev/src/Npgsql.NodaTime/TimestampHandler.cs#L19